contrib/tablefunc/tablefunc--1.1.sql

```diff
@@ -0,0 +1,88 @@
+/* contrib/tablefunc/tablefunc--1.1.sql */
```

Comment:
Do we really need this file? According to [Postgres 12 documentation](https://www.postgresql.org/docs/12/extend-extensions.html): 
> if you have released a foo extension in versions 1.0, 1.1, and 1.2, there should be update scripts foo--1.0--1.1.sql and foo--1.1--1.2.sql. Before PostgreSQL 10, it was necessary to also create new script files foo--1.1.sql and foo--1.2.sql that directly build the newer extension versions, or else the newer versions could not be installed directly, only by installing 1.0 and then updating. That was tedious and duplicative, but now it's unnecessary

so I guess having 'tablefunc--1.0--1.1.sql' should be enough...

Reply:
Fixed